examples/gym/train_ppo_gym.py

```diff
@@ -0,0 +1,180 @@
+        super().__init__(self.pi, self.v)
+
+    def pi_and_v(self, state):
+        return self.pi(state), self.v(state)
+
+
+class A3CFFMellowmax(chainer.ChainList, a3c.A3CModel):
+    """An example of A3C feedforward mellowmax policy."""
+
+    def __init__(self, ndim_obs, n_actions, hidden_sizes=(200, 200)):
+        self.pi = policies.MellowmaxPolicy(
+            model=links.MLP(ndim_obs, n_actions, hidden_sizes))
+        self.v = links.MLP(ndim_obs, 1, hidden_sizes=hidden_sizes)
+        super().__init__(self.pi, self.v)
+
+    def pi_and_v(self, state):
+        return self.pi(state), self.v(state)
+
+
+class A3CFFGaussian(chainer.ChainList, a3c.A3CModel):
+    """An example of A3C feedforward Gaussian policy."""
+
+    def __init__(self, obs_size, action_space,
+                 n_hidden_layers=2, n_hidden_channels=200):
+        hidden_sizes = (n_hidden_channels,) * n_hidden_layers
+        self.pi = policies.FCGaussianPolicyWithStateIndependentCovariance(
+            obs_size, action_space.low.size,
+            n_hidden_layers, n_hidden_channels,
+            var_type='diagonal', nonlinearity=F.tanh,
+            bound_mean=True,
+            min_action=action_space.low, max_action=action_space.high)
+        self.v = links.MLP(obs_size, 1, hidden_sizes=hidden_sizes)
+        super().__init__(self.pi, self.v)
+
+    def pi_and_v(self, state):
+        return self.pi(state), self.v(state)
+
+
+def main():
+    import logging
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--gpu', type=int, default=None)
+    parser.add_argument('--env', type=str, default='CartPole-v1')
+    parser.add_argument('--arch', type=str, default='FFSoftmax',
+                        choices=('FFSoftmax', 'FFMellowmax',
+                                 'FFGaussian'))
+    parser.add_argument('--seed', type=int, default=None)
+    parser.add_argument('--outdir', type=str, default=None)
+    parser.add_argument('--steps', type=int, default=10 ** 5)
+    parser.add_argument('--eval-interval', type=int, default=2048)
+    parser.add_argument('--eval-n-runs', type=int, default=10)
+    parser.add_argument('--reward-scale-factor', type=float, default=1e-2)
+    parser.add_argument('--render', action='store_true', default=False)
+    parser.add_argument('--lr', type=float, default=3e-4)
+    parser.add_argument('--weight-decay', type=float, default=0.0)
+    parser.add_argument('--demo', action='store_true', default=False)
+    parser.add_argument('--load', type=str, default='')
+    parser.add_argument('--logger-level', type=int, default=logging.DEBUG)
+    parser.add_argument('--monitor', action='store_true')
+
+    parser.add_argument('--update-interval', type=int, default=2048)
+    parser.add_argument('--batchsize', type=int, default=64)
+    parser.add_argument('--epochs', type=int, default=10)
+    args = parser.parse_args()
+
+    logging.getLogger().setLevel(args.logger_level)
```

Comment:
This should be `logging.basicConfig(level=args.logger_level)`, otherwise this example code will output nothing on screen, even the paths to the output directory.